README.md

```diff
@@ -16,7 +16,7 @@ conda init
 ```
 Create an enviroment `aiida1` with python 3.6 and activate this new environment
 ```
-conda create -n aiida1 python=3.6 -y
```

Comment:
Have you tested this?
I think one usually first does the `conda create` command, and then the `conda install -c conda-forge ...`

Reply:
Thanks for @ltalirz. Yes, that would be better.